src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py

```diff
@@ -486,6 +490,13 @@ async def do_delete(self, job, release_name):
         job.set_progress(100, f'{release_name!r} chart release deleted')
         return True
 
+    @private
+    def helm_file_args(self, chart_path):
+        file_args = ['-f']
+        if os.path.exists(os.path.join(chart_path, 'ix_values.yaml')):
+            file_args = [os.path.join(chart_path, 'ix_values.yaml'), '-f']
+        return file_args
```

Comment:
I meant more, all between context manager for tempfile, write config and then run command seem to be the same. Only changing the helm parameter (install vs upgrade)